apps/web/src/components/modals/alertModal.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState, type FunctionComponent } from 'react';
+import {
+    AlertDialog,
+    AlertDialogAction,
+    AlertDialogCancel,
+    AlertDialogContent,
+    AlertDialogDescription,
+    AlertDialogFooter,
+    AlertDialogHeader,
+    AlertDialogTitle,
+    AlertDialogTrigger,
+} from "~/components/ui/alert-dialog"
+import { Input } from '../ui/input';
+type AlertModalProps = {
+    isOpen: boolean;
+    onOpenChange: (open: boolean) => void;
+    title: string;
+    description: string;
+    unlockString?: string;
+    onCancel: () => void;
+    onConfirm: () => void;
+};
+
+export const AlertModal: FunctionComponent<AlertModalProps> = ({ isOpen, onOpenChange, title, description, unlockString, onCancel, onConfirm, }) => {
```

Comment:
This should be changed to:
```suggestion
export function AlertModal({ isOpen, onOpenChange, title, description, unlockString, onCancel, onConfirm, }: AlertModalProps) {
```

Reply:
already fixed